src/mito2/src/test_util.rs

```diff
@@ -146,6 +151,36 @@ impl TestEnv {
         MitoEngine::new_for_test(config, logstore, object_store_manager, manager, listener)
     }
 
+    pub async fn create_engine_with_multiple_object_stores(
+        &mut self,
+        config: MitoConfig,
+        manager: Option<WriteBufferManagerRef>,
+        listener: Option<EventListenerRef>,
+        custom_storage_types: Vec<StorageType>,
```

Comment:
suggestion: We don't need to expose `StorageType` in these test cases. We can pass a slice of strings/strs, e.g. `custom_storage_types: &[&str]` since we only need a list of names for custom storages.